<launch>
  <!-- The end-effector frame_id -->
  <arg name="ee_frame"/>

  
  <!-- Bring up the visp auto tracker -->
  <include file="$(find visp_auto_tracker)/launch/tracklive_usb.launch">
  </include>

  <!-- Calibrate the extrinsics for a kinect mounted to a robot base -->
  <include file="$(find kinect_bhand_calib)/launch/kinect_bhand_calib.launch">

    <arg name="marker_parent_frame" value="/left_wam/base_link"/>
    <arg name="camera_parent_frame" value="/left_wam/wrist_palm_link"/>
    <arg name="camera" value="/camera/rgb"/>

    <arg name="camera_frame" value="/camera_rgb_optical_frame"/>
    <arg name="xyz_optical_base" value="[0.000, 0.000, 0.000]"/>
    <arg name="rpy_optical_base" value="[-1.571, -0.000, -1.571]"/>
  </include>



</launch>
